test/e2e/antreapolicy_test.go

```diff
@@ -273,6 +275,40 @@ func testInvalidACNPTierDoesNotExist(t *testing.T) {
 	}
 }
 
+func testInvalidACNPPortRangePortUnset(t *testing.T) {
+	invalidNpErr := fmt.Errorf("invalid Antrea ClusterNetworkPolicy egress rule ports set with endPort but no Port accepted")
```

Comment:
I find the error message confusing, especially the "ports" in the middle.

Don't you just mean "invalid Antrea ClusterNetworkPolicy egress rule with endPort but no Port accepted"?

same for the other error messages

Reply:
done in all places